src/pages/Components/TextInput/index.md

```diff
@@ -7,52 +7,53 @@ order: 2
 
 > Inputs are used to **create input fields with text on a single line**. Their states depends on the user interaction or the context.
 
-<preview path="src/pages/Components/TextInput/previews/input-preview"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-preview"></preview>
 
 ## Standard input states :
 
 ### Hover
 
 When the input is hovered due to user navigation.
 
-<preview path="src/pages/Components/TextInput/previews/input-hover-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-hover-state"></preview>
 
 ### Focus
 
 When the input is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
 
-<preview path="src/pages/Components/TextInput/previews/input-focus-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-focus-state"></preview>
 
 ### Disabled
 
 When the input field is inactive or inaccessible. It is usually used when the input, for some reason, is forbidden or blocked to be edited.
 
-<preview path="src/pages/Components/TextInput/previews/input-disabled-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-disabled-state"></preview>
 
 ### Valid
 
 When the entry is correct against the criteria determined upstream.
 
-<preview path="src/pages/Components/TextInput/previews/input-valid-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-valid-state"></preview>
 
 ### Invalid
 
 When the entry is invalid against the criteria determined upstream.
 
-<preview path="src/pages/Components/TextInput/previews/input-invalid-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-invalid-state"></preview>
 
-## Do's and don'ts
+## Using a left Icon :
 
-<hintitem>
-You can cumulate différents states like that :
-</hintitem>
-<preview path="src/pages/Components/TextInput/previews/input-invalid-event"></preview>
+Use a left icon to inform about a specific information type or format you expect the user to provide.
+
+<preview nude="true" path="src/pages/Components/TextInput/previews/left-icon-input"></preview>
```

Comment:
You have to put the icon in 32px for the hierarchy and the icon visibility. We always have complexer icons to the left than the right.
Moreover, the right icon has 24px, so if the left icon has 24px too, they cancel themselves.

Reply:
32px is enormous, I think we should use the 16px icon on the validation instead.

Reply:
I think what's important at the validation step, is the validation, plus the color make the hierarchy for you. even tiny red will always stand out. 

But I would definitely use the 16px icon on the validation. 

what do you think ?

Reply:
I did some tests in sketch.
Ok to keep the left icon in 24px, and put the right icon in 16px but we must use the icon *Navigation_Notification_Check_16px* for the valid state and the *Navigation_Display_Close_16px* for the error state.

Reply:
like props in the mineral line 🤩

Reply:
done

Reply:
top